Dockerfile

```diff
@@ -9,7 +9,6 @@ ENV CYPRESS_INSTALL_BINARY=0
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1
 
 RUN useradd -m -d /frontend redash
-USER redash
```

Comment:
Ahhh.  Did you run into the same weird permissions problem in the Dockerfile when building the image with the updated docker package too?

Reply:
yes, I'll delete this from this PR if the E2E tests pass